skeleton/package.json

```diff
@@ -18,13 +18,13 @@
     "params": "pure_funcs=\"F2,F3,F4,F5,F6,F7,F8,F9,A2,A3,A4,A5,A6,A7,A8,A9\",pure_getters,keep_fargs=false,unsafe_comps,unsafe"
   },
   "devDependencies": {
-    "elm": "^0.19.0-bugfix6",
+    "elm": "^0.19.0-no-deps",
```

Comment:
Never heard of this, is it a new thing?

Reply:
That's the latest stable version available from npm.

![](https://i.imgur.com/uOISc6e.png)